### ISSUE TITLE
How to train a subset of COCO

### ISSUE THREAD

**Issue by whstarxu54** [matterport/Mask_RCNN#104]
Thank you for your wonderful work on implementing Mask R-CNN. I have succeeded in training and testing on the COCO 2014 dataset. 
Now, how can I train a model on a subset of COCO, e.g., I only want to train a model that only does segmentation on "person" and "dog"?

**Comment by waleedka** [comment]
The `load_coco()` function has a class_id parameter that allows you to pick which subset of classes to load. You'll also need to change the `NUM_CLASSES` attribute in the config class.

**Comment by whstarxu54** [comment]
@waleedka Thank you. I have successfully trained a subset after making the modifications you mentioned.

**Comment by mrzw** [comment]
@whstarxu54 Excuse me,  Have you successfully trained the model on COCO dataset with the mAP in the paper of Mask R-CNN. Besides, how can I train a subset of COCO like you with the trained model mask_rcnn_coco.h5.  Just change the NUM_CLASSES,  the class_id parameter in load_coco() function, and load weights exclude ["mrcnn_class_logits", "mrcnn_bbox_fc",  "mrcnn_bbox", "mrcnn_mask"] is OK?

**Comment by matiqul** [comment]
But I want to train my own dataset with own classes like MSCOCO dataset.......

**Comment by endluo** [comment]
nice job, but i want to train muiltiple-class，as  car,motobike,bicycle on the coco ，what should i do？thx
